doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,256 @@
+# DR014 Management Policy responses
+
+- **Status:** Proposed
+- **Driver:** @foundrytom
+- **Approver:** @feltech @ecmorris @antirotor @mattdaw
+- **Outcome:** A Manager should additionally imbue a `managementPolicy`
+  response with the traits that it is capable of resolving/persisting
+  for each trait set.
+
+## Background
+
+The ability for a host to query a manager's capabilities and intent is a
+critical part of the OpenAssetIO abstraction.
+
+OpenAssetIO classifies entities (assets) using a Trait Set, which lists
+one or more traits that describe it's nature. For example, and image
```

Comment:
"its nature"
Github doesn't have [Gitlab's "insert suggestion" feature](https://docs.gitlab.com/ee/user/project/merge_requests/reviews/suggestions.html)? I feel bad being this pedantic without it.

Reply:
Thanks! If you switch to "Files Changed" view, you should be able to [do the suggestions thing](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request).

Reply:
Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/792/commits/59ec5fc906e84b5630641d7564614939a25f37cf) (with others).